Add documentation about the build cache (#44)
cloudwatt/nixpkgs-tungsten: README.md

$ scons contrail-control
```

#### Build cache

> In the case of C++ artifacts the build can be quite long. Instead of running a

Reviewer: ```suggestion
Building C++ dependencies can be quite time consuming. Instead of running a
```

> shell for `contrail50.control` you can use `contrail50.dev.control` attribute

Reviewer: ```suggestion
shell for `contrail50.control` you can use the `contrail50.dev.control` attribute
```
